tests/Api/RenderTest.php

```diff
@@ -15,8 +15,7 @@ public function testTemplateNotFound(): void {
     $templateKey = __FUNCTION__;
     $dataKey = __FUNCTION__;
 
-    $response = $this->get($this->buildURI($templateKey, $dataKey));
-    Assert::assertEquals(404, $response->getStatusCode());
+    $this->assertStatus(404, $this->get($this->buildURI($templateKey, $dataKey)));
```

Comment:
i really hate this `buildURI` function ...  whose definition is not even in this file ...in an API test it is important for the reader to be able to see what the dang endpoint is lol

I get the convenience of the abstraction ... but it isn't worth it

Reply:
I got ya, It's still there from last time. I'll fix it.